app/metrics.yaml

```diff
@@ -254,3 +260,249 @@ control:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-11-01"
+
+pages:
+  page_load:
+    type: timing_distribution
+    description: >
+      Measuring how long each page takes to load.
+    time_unit: nanosecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+
+immersive:
+  duration:
+    type: timing_distribution
+    description: >
+      Measuring how long users stay in the immersive mode.
+    time_unit: microsecond
```

Comment:
Not really a problem, just curiosity: I suspect `pages.page_load` and this one have about the same `time_unit`? Or are you really expecting data to have different magnitudes, here?

Reply:
I expect the immersive mode measurement needs different magnitudes. People usually enter the immersive mode for watching 360 videos and interactive games. It would be very high chance to beyond the 10 mins limitation of using `time_unit: nanosecond`.

Reply:
Got it, thanks for clarifying!